player.cpp

```diff
@@ -12,10 +16,12 @@ void test_player() {
     // Here write tests for players
     // 17 - The player has a score counting system
     const player player_g;
-    const int expected_player_score = 0;
-    const int player_score = player_g.get_score();
+    const int expected_player_score{0};
+    const int player_score{player_g.get_score()};
     //check player score
     assert(expected_player_score == player_score);
-
-
+    const double x{player_g.get_x()};
+    const double y{player_g.get_y()};
```

Comment:
That's a very smart, concise way to ensure get_x returns a double!

Reply:
Haha, we knew :sunglasses: